### ISSUE TITLE
I suspect 1:1 calls should look like a normal 1:1 call

### ISSUE THREAD

**Issue by ara4n** [vector-im/matrix-video-chat#196]
i.e. fullscreen for the speaker, and small thumbnail for local video, rather than the massive local video we have today.

**Comment by SimonBrandner** [comment]
Personally, I believe that if this is to be a video-conferencing app the UI should mostly reflect that and shouldn't change way too much depending on the number of users. I would find that inconsistent and potentially confusing

**Comment by babolivier** [comment]
Also the current UI (especially in "Speaker" mode) works really well on a 1:1 call when screensharing

**Comment by robertlong** [comment]
Agreed! Waiting on designs. Closing as this is a duplicate of #44
